akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -106,7 +106,7 @@ class CircuitBreaker(
   scheduler:                Scheduler,
   maxFailures:              Int,
   callTimeout:              FiniteDuration,
-  resetTimeout:             FiniteDuration,
+  val resetTimeout:         FiniteDuration,
```

Comment:
in case I can rebind `resetTimeout` to something like:

```scala
val originalResetTimeout = resetTimeout
```
in the class body, this should solve the binary compatibility right?